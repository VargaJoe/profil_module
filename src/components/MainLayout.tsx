import * as React				from 'react';

class MainLayout extends React.Component<any, any> {
    render() {

		return (
			<div>
				MainLayout
			</div>
		);
	}
}

export default MainLayout;
 