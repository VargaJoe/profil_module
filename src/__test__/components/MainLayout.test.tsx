import * as React 						  	from 'react';
import MainLayout 							from '../../components/MainLayout';
import { 
	configure, 
		mount } 							from 'enzyme';
import * as Adapter 					  	from 'enzyme-adapter-react-16';
import toJson 				      			from 'enzyme-to-json';

configure( {adapter: new Adapter()} );

describe('<MainLayout /> shallow rendering', () => {
		
	const mainLayout = mount(<MainLayout />); 
		
	// test Snapshot 
	it('Match to snapshot', () => {
		expect(toJson(mainLayout)).toMatchSnapshot();
	});
		
});
 