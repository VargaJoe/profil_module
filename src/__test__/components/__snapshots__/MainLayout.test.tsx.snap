// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainLayout /> shallow rendering Match to snapshot 1`] = `
<div>
  MainLayout
</div>
`;
